AccountMasterKey.js

```diff
@@ -158,24 +159,32 @@ export class AccountMasterKey {
    * @param {Object} options - The options to use.
    * @param {string} options.accountId - The ID of the account associated with
    *   this master key.
-   * @param {Object} options.kmsService - The kmsService to use to perform key
+   * @param {Object} [options.service] - The kmsService to use to perform key
```

Comment:
Need to keep this the same or it's a breaking change.

```suggestion
   * @param {Object} [options.kmsService] - The kmsService to use to perform key
```